### ISSUE TITLE
Template misplacement for Visual Studio 2015.

### ISSUE THREAD

**Issue by nilllzz** [mono/MonoGame#4063]
Hello there,

the MonoGame installer misplaces the templates for Visual Studio 2015 (full release).
The correct output folder is 

C:\Users\user\Documents\\**Visual Studio 14**\Templates\ProjectTemplates\Visual C#

And MonoGame installs to

C:\Users\user\Documents\\**Visual Studio 2015**\Templates\ProjectTemplates\Visual C#

This causes the templates to not show up, even when Visual Studio 2015 is selected in the installer.

**Comment by BrianPeek** [comment]
Hm...are you on the final version of VS2015?  I have a Visual Studio 2015 directory, but no Visual Studio 14 directory.  Did you have a pre-release version installed previously?

**Comment by nilllzz** [comment]
I do have the final version of 2015, I installed it yesterday. And yes, I had a preview installed some time ago.
It is really bizarre that when a 14 folder exist that Visual Studio starts using that instead of the usual 2015, without telling me.

**Comment by Nezz** [comment]
The correct folder is `Visual Studio 2015`. The `Visual Studio 14` name was used in the CTP releases.

**Comment by nilllzz** [comment]
Sure, but I deinstalled the preview version a long time ago, and just now installed the full release.
I kept the Visual Studio 14 directory in my Documents folder though, and apparently, when this folder exists and you install Visual Studio 2015, it detects this and uses this folder, despite being the full release.

So when I installed MonoGame, it put the templates into a new Visual Studio 2015 folder, and VS couldn't detect them because it was using the Visual Studio 14 folder.

I found a setting to reverse the directory assignment from 14 to 2015, so I guess this is closed.
